website_event_crm_invitation/models/event_type.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Tecnativa - Jairo Llopis
+# Copyright 2025 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+from odoo.osv import expression
+
+
+class EventType(models.Model):
+    _inherit = "event.type"
+
+    def _published_events_domain(self):
+        """Get domain for open and published events of this category."""
+        domain = self._events_domain()
+        domain += [
+            ("date_begin", ">", fields.Datetime.now()),
+            ("website_published", "=", True),
+            ("seats_limited", "=", False),
+        ]
+        # Search limited events with available seats (not storable)
+        events_with_seats = self.env["event.event"].search(
+            [
+                ("seats_limited", "=", True),
+                ("date_begin", ">", fields.Datetime.now()),
+                ("website_published", "=", True),
+            ]
+        )
```

Comment:
Please consider the base domain here too. Otherwise, it retrieves events from any event type.
I created a new event type without any events, and I got events from other event types, which I think is incorrect.

```suggestion
        base_domain = self._events_domain()
        domain = base_domain + [
            ("date_begin", ">", fields.Datetime.now()),
            ("website_published", "=", True),
            ("seats_limited", "=", False),
        ]
        # Search limited events with available seats (not storable)
        events_with_seats = self.env["event.event"].search(
            base_domain + [
                ("seats_limited", "=", True),
                ("date_begin", ">", fields.Datetime.now()),
                ("website_published", "=", True),
            ]
        )
```

Reply:
done, thanks